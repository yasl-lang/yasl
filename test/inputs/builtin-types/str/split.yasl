echo 'the quick brown fox'->split(' ')
echo 'the quick brown fox'->split(' ')
echo 'the quick brown fox'->split('0')
echo ' a b c         d     e         '->split()
echo '               '->split()
echo 'theaaquickaabrownaafox'->split('aa')
echo 'theaaquickaabrownaafox'->split(1)
echo '     a      b c      d e     f g    h  '->split(4)
echo '     a      b c      d e     f g    h  '->split(0)
echo 'abcdef'->split()
echo 'abcdef'->split(1)
echo 'a b c d e f'->split(0)
echo 'abcdefghijk'->split(2)
echo 'a b c d e f g h'->split(' ')
echo 'a b c d e f g h'->split(' ', 0)
echo 'abababab'->split('b', 2)
const y = '1 2 3 4 5'
const x = 'a b c e d'
echo y->split(0)
echo y->split(1)
echo y->split(2)
echo x->split(' ', 0)
echo x->split(' ', 1)
echo x->split(' ', 2)
echo ''->split()
echo ''->split(' ')
