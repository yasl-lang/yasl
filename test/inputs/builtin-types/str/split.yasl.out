[the, quick, brown, fox]
[the, quick, brown, fox]
[the quick brown fox]
[a, b, c, d, e]
[]
[the, quick, brown, fox]
[theaaquickaabrownaafox]
[a, b, c, d, e     f g    h  ]
[     a      b c      d e     f g    h  ]
[abcdef]
[abcdef]
[a b c d e f]
[abcdefghijk]
[a, b, c, d, e, f, g, h]
[a b c d e f g h]
[a, a, abab]
[1 2 3 4 5]
[1, 2 3 4 5]
[1, 2, 3 4 5]
[a b c e d]
[a, b c e d]
[a, b, c e d]
[]
[]
